.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -142,7 +142,8 @@ jobs:
             --azure-resource-group ${{ env.name }} \
             --wait=false \
             --rollback=false \
-            --config monitor-aggregation=none"
+            --config monitor-aggregation=none" \
+            --base-version=v1.10.5
```

Comment:
@tklauser Would it be better to use `v1.10.0`, or perhaps we can use just `v1.10`? Do we have a scenario where the minor version is important? I'm worried we'd need to update this regularly.

Reply:
if I set it to v1.10, I get `Creating ConfigMap for Cilium version 1.10.0..` in the logs :/

Reply:
Yes, let's use either `v1.10` or `v1.10.0` as the base version. `cilium-cli` is only interested in the minor version. I think `v1.10` should work as well from skimming through the code, but I didn't try it out. If `--base-version=v1.10` works, I'd prefer that one over `v1.10.0`.

Reply:
> if I set it to v1.10, I get `Creating ConfigMap for Cilium version 1.10.0..` in the logs :/

I think that's fine. As mentioned in my comment above, the patch version doesn't matter for cilium-cli.